### DISCUSSION TITLE
E-stop stop to work correctly

### DISCUSSION THREAD

**Discussion by rmaltais** [grblHAL/core#499] [Q&A]
Hello all,

I have a strange problem. Since I updated new firmware, my E-stop no longer has any effect and it no longer produces alarm 10. However, I can see very clearly the signal on the E software which turns red. I had to redo the configuration but I don't see what could be causing this problem. I compiled my new firmware on the WEB interface. I have a this controler:

![msedge_Sb7w9TvTHd](https://github.com/grblHAL/core/assets/41405404/7a8105e6-e450-4c17-934c-281cc0bf358d)

thank

**Comment by karoria** [comment]
Do you connect Estop to reset pin?  If not connect it to reset pin.

**Reply by rmaltais** [reply]
Sure it's connect. The problem is in the firmware, i change the firmware and now it's work. The last firmware did not work with my controller. Now it's work but i have an other problem with probe. The probe stay freeze after triggered. 

Not very better.

**Reply by terjeio** [reply]
There was a regression in control signals IRQ handling, should be fixed now.

**Reply by rmaltais** [reply]
Ok work fine now but probe still stop after first triggered. The state of probing still always in "probing..." and no move to retract.
